packages/atomic/src/components/search/result-template-components/atomic-quickview/e2e/atomic-quickview.e2e.ts

```diff
@@ -1,27 +1,113 @@
+import {DEFAULT_MOBILE_BREAKPOINT} from '../../../../../utils/replace-breakpoint';
 import {test, expect} from './fixture';
 
+function mobileViewportSize() {
+  return {
+    width: parseInt(DEFAULT_MOBILE_BREAKPOINT.slice(0, -2)) - 1,
+    height: 1080,
+  };
+}
+
+function desktopViewportSize() {
+  return {
+    width: parseInt(DEFAULT_MOBILE_BREAKPOINT.slice(0, -2)) + 1,
+    height: 1080,
+  };
+}
+
 test.describe('Quickview', () => {
-  test.beforeEach(async ({quickview}) => {
-    await quickview.load();
-  });
+  test.describe('when displaying search results', () => {
+    test.beforeEach(async ({quickview}) => {
+      await quickview.load();
+    });
 
-  test('should render the quickview button', async ({quickview}) => {
-    await expect(quickview.resultButton).toBeVisible();
+    test('should render the quickview button', async ({quickview}) => {
+      await expect(quickview.resultButton).toBeVisible();
+    });
   });
 
   test.describe('when the quickview button is clicked', () => {
     test.beforeEach(async ({quickview}) => {
+      await quickview.load();
       await quickview.resultButton.click();
+      await quickview.modal.waitFor({state: 'visible'});
+    });
+
+    test('should be A11y compliant', async ({makeAxeBuilder}) => {
+      const accessibilityResults = await makeAxeBuilder().analyze();
+      expect(accessibilityResults.violations).toEqual([]);
     });
 
     test('should display the quickview modal', async ({quickview}) => {
       await expect(quickview.modal).toBeVisible();
     });
 
+    test('should display a clickable link in the header', async ({
+      quickview,
+    }) => {
+      await expect(quickview.header).toBeVisible();
+      await expect(quickview.titleLink).toBeVisible();
+      await expect(quickview.titleLink).toHaveAttribute('href', /.+/);
```

Comment:
Alternatively you could do a click and check that the page does load with page.waitForURL()